src/docs/self-hosted/index.mdx

```diff
@@ -107,6 +107,31 @@ By default Sentry sends anonymous usage statistics to the Sentry team. It helps
 
 You can disable this feature by adding `SENTRY_BEACON = False` into `sentry.conf.py` file.
 
+### Errors-only Self-hosted Sentry
+
+This feature is experimental and now in beta. Starting from 24.8.0+ onwards, users will have the ability to choose between two distinct types of self-hosted Sentry deployments, tailored to their specific needs and preferences.
```

Comment:
nit: first sentence should be its own bolded paragraph: "This is an experimental alpha feature. Use at your own risk!"